microSALT/utils/pubmlst/constants.py

```diff
@@ -1,17 +1,42 @@
 from enum import Enum
 from werkzeug.routing import Map, Rule
 
+def add_prefix_to_rules(url_map: Map, prefix: str):
+    """
+    Add a prefix to all rules in the given URL map.
+
+    :param url_map: The original URL map.
+    :param prefix: The prefix to add (e.g., '/api').
+    :return: A new URL map with the prefix applied.
+    """
+    new_rules = []
```

Comment:
Seems this could be simplified to:
```python
    new_rules = [
        Rule(
            f"{prefix}{rule.rule}",
            endpoint=rule.endpoint,
            methods=rule.methods,
        )
        for rule in url_map.iter_rules()
    ]
    return Map(new_rules)
```